drivers/hud_path_report/app/models/hud_path_report/generators/fy2024/base.rb

```diff
@@ -348,11 +353,21 @@ def initialize(generator = nil, report = nil, options: {})
         group(:DateProvided).pluck(:DateProvided, Arel.sql(array_agg(s_t[:TypeProvided]).to_sql)).to_h
     end
 
+    private def cmh_service_provided(enrollment)
+      enrollment.services.path_service.between(start_date: @report.start_date, end_date: @report.end_date).
+        pluck(Arel.sql(s_t[:TypeProvided].to_sql)).include?(4)
```

Comment:
You shouldn't need the arel to do this pluck since it's just pulling the column from the services table
```suggestion
        pluck(:TypeProvided).include?(4)
```

Reply:
These are also an N+1 but I guess given the usual size of PATH projects and PATH reports, that's probably ok and seems to be what we're doing with `path_services` and `path_referrals` so let's leave it like that for now.